protocol-specs/generators/multisignature_registration/index.js

```diff
@@ -14,164 +14,206 @@
 
 'use strict';
 
-const {
-	hexToBuffer,
-	intToBuffer,
-	hash,
-	bufferToHex,
-	signData,
-} = require('@liskhq/lisk-cryptography');
+const { signData } = require('@liskhq/lisk-cryptography');
+const { Codec } = require('@liskhq/lisk-codec');
```

Comment:
Why to use class and not the instance `codec`?

Reply:
I just wanted to make sure there is no duplicate registration of schema